partcad/src/partcad/builtin/render/render_svg.py

```
@@ -59,31 +82,27 @@ def _normalize_mesh(shape):
     try:
         wrapped = request["wrapped"]
-        if request.get("normalize_mesh"):
+        if needs_mesh_normalization(request):
             wrapped = _normalize_mesh(wrapped)
 
         b3d_obj = b3d.Solid.make_box(1, 1, 1)
         b3d_obj.wrapped = wrapped
 
-        viewport_origin = tuple(request.get("viewport_origin"))
-        viewport_up = tuple(request.get("viewport_up", [0, 0, 1]))
         visible, hidden = b3d_obj.project_to_viewport(
```

Ruff (0.16.1)

[warning] Unpacked variable `hidden` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)